cds_ils/importer/importer.py

```diff
@@ -243,26 +247,22 @@ def import_summary(self):
             "series": self.series_list,
             "raw_json": self.json_data,
             "document_json": doc_json,
-            "document": self.document
+            "document": self.document,
         }
 
     def preview_delete(self):
         """Preview deleting a record."""
         self._validate_provider()
-        self.action = "update"
+        self.action = "none"
         # finds the exact match, update records
         self.document = self._match_document()
 
         if self.document:
             self.output_pid = self.document["pid"]
             self.eitem_summary = \
                 self.eitem_importer.preview_delete(self.document)
-            document_has_only_serial_relations = \
-                len(self.document.relations.keys()) \
-                and 'serial' in self.document.relations.keys()
-            if not self.document.has_references() \
-                    and document_has_only_serial_relations:
-                self.action = "delete"
+            self.preview_delete_document(self.document)
+            self.action = "delete"
```

Comment:
@kprzerwa is it fine to just assign the action here regardless? I saw there were some conditions before that were applied in order to do that.

Reply:
In the delete we try to delete even if the condition is not fulfilled (probably raising an error). As the results between the preview and the real delete must be the same, the behaviour here was changed to meet that objective.